style.css

```diff
@@ -687,10 +708,6 @@ fieldset {
 }
 
 @media screen and (max-width: 470px) {
```

Comment:
Why don't we make the popup window more responsive for mobile view 👍 

> Your design
![Screenshot (293)](https://user-images.githubusercontent.com/66112760/141310434-e493819a-bd9b-4f90-8920-d4a67df38cb8.png)

> Figma design

![Screenshot (292)](https://user-images.githubusercontent.com/66112760/141310485-fdca3e12-53d5-4478-88c0-b756af7ab07e.png)